internal/lib/remove.go

```diff
@@ -20,17 +20,22 @@ func (c *ContainerServer) Remove(ctx context.Context, container string, force bo
 	cStatus := ctr.State()
 	switch cStatus.Status {
 	case oci.ContainerStatePaused:
-		return "", errors.Errorf("cannot remove paused container %s", ctrID)
+		if !force {
+			return "", errors.Errorf("cannot remove paused container %s", ctrID)
+		}
+		if err = c.runtime.UnpauseContainer(ctx, ctr); err != nil {
+			return "", errors.Wrapf(err, "unable to unpause container %s", ctrID)
+		}
```

Comment:
If we're calling unpause in StopContainer, we can merge these two cases
```
	case oci.ContainerStateCreated, oci.ContainerStatePaused, oci.ContainerStateRunning:
	    if !force {
                return ...
            }
            c.StopContainer(...)
        }
```

Reply:
fixed